README.hpcgap.md

```diff
@@ -0,0 +1,14 @@
+# HPC-GAP
+
+The beta release of GAP 4.9 includes experimental code to support multithreaded
+programming in GAP. GAP and HPC-GAP codebases diverged during the project, and
```

Comment:
This text seems to assume the reader is already familiar with the term HPC-GAP. Perhaps:

> ... programming in GAP, dubbed HPC-GAP (where HPC is short for `high performance computing`).